docs/contributing.md

```diff
@@ -159,6 +159,50 @@ To use a preview build for a package within a project, you need to override the
 
 4. Run `yarn install`.
 
+## Adding new packages
+
+> If you're migrating an existing package to the monorepo, please see [the package migration documentation](./package-migration-process-guide.md).
+> You may be able to make use of `create-package` when migrating your package, but there's a lot more to it.
+
+Manually a new monorepo package can be a tedious, even frustrating process. To spare us from that
+suffering, we have created a CLI that automates most of the job for us, creatively titled
+[`create-package`](../scripts/create-package/). To create a new monorepo package, follow these steps:
+
+1. Create a new package using `yarn create-package`.
+   - Use the `--help` flag for usage information.
+   - Once this is done, you can find a package with your chosen name in `/packages`.
+2. Make sure your license is correct.
+   - By default, `create-package` gives your package an MIT license.
+   - If your desired license is _not_ MIT, then you must update your `LICENSE` file and the
+     `license` field of `package.json`.
+3. Add your dependencies.
+   - Do this as normal using `yarn`.
+   - Remember, if you are adding other monorepo packages as dependents, don't forget to add them
+     to the `references` array in your package's `tsconfig.json` and `tsconfig.build.json`.
+
+And that's it!
+
+### Contributing to `create-package`
+
+Along with this documentation, `create-package` is intended to be the source of truth for the
+process of adding new packages to the monorepo. Consequently, to change that process, you will want
+to change `create-package`.
```

Comment:
The way I see it is that the module template is a rough source of truth for this repo and packages therein, but it's not as 1-to-1 as it is for polyrepos (the structure of a package in this repo doesn't match the module template, for instance, and some of the GitHub workflows and package scripts are a bit different than the module template). We should seek to follow the module template where we can, in other words, but there are some differences that will always exist due to being a monorepo and not a polyrepo. Perhaps we ought to document these differences somewhere, would that be helpful?

Reply:
Is this an action item for this PR?
> So I agree with documenting where create-package sits when considering the module template but perhaps we ought to document these differences in the context of the entire monorepo somewhere. Would that be helpful?

Reply:
What we have here seems to make sense to me — there is no mention of the module template, so we can reasonably assume that the source of truth is local to this repo, not the module template. However, I also realize that the module template _is_ the source of truth for everything else, so if we are trying to connect the dots here, then perhaps we add a blurb to the module template saying that it's more of a source of truth for polyrepos than monorepos?

Reply:
Not to dismiss your suggestion @MajorLift but maybe it's something we can add in a future PR since this one has been around for a bit? Are you okay with that?